airflow/www/templates/airflow/graph.html

```diff
@@ -258,5 +270,22 @@
         return false
     }
 
+    function zoomWithSlider() {
+        var svg = d3.select("body").select("svg");
+        var container = svg.select("g");
+        var scale = 0.5 + parseInt(slider.getValue())/10;
+
+        container.attr("transform", "translate(20, 20) scale(" + scale + ")");
+
+        // var h = svg.attr("height"), w = parseInt(svg.style("width"));
```

Comment:
The commented out code here is for zooming with the centre of the svg as a focus point. In testing, this seems to be less useful then simply zooming at "20, 20" as things tends to fly off screen.

Would be interested in your input @mistercrunch 

Let me know what you'd prefer and I can make the changes.